mlflow/plural/recipes/mlflow-aws.yaml

```diff
@@ -25,9 +25,5 @@ sections:
   items:
   - type: TERRAFORM
     name: aws
-  - type: TERRAFORM
-    name: aws-standalone
   - type: HELM
     name: mlflow
```

Comment:
This means existing users with an mlflow install are going to have the operator uninstalled and this chart replacing it?

Reply:
That is the idea. MLFlow standalone will be replaced by this, the operator is no longer needed either.